### ISSUE TITLE
Fin de l'authentification HTTP Basic et OAuth 1.0 sur osm.org

### ISSUE THREAD

**Issue by flacombe** [osm-fr/infrastructure#531]
Je relaye cette information ici
https://community.openstreetmap.org/t/oauth-1-0a-and-http-basic-auth-shutdown/108490

* Fin de HTTP Basic et OAuth 1.0
* Remplacement par OAuth 2.0

Peut-être que cela a un impact sur différents services

Bonne après-midi

**Comment by cayenne17** [comment]
comme application OSM-FR impacté, je vois : 
- https://commerces.openstreetmap.fr
- https://projetdumois.fr
- http://umap.openstreetmap.fr/
- http://osmose.openstreetmap.fr/

**Comment by flacombe** [comment]
PDM a été prévenu via cet autre ticket
https://github.com/vdct/ProjetDuMois/issues/312

**Comment by cayenne17** [comment]
umap prévenu : https://github.com/umap-project/umap/issues/1279https://github.com/umap-project/umap/issues/1279

**Comment by frodrigo** [comment]
Osmose aussi concerné, et toujours pas en v2
https://github.com/osm-fr/osmose-frontend/issues/481

**Comment by frodrigo** [comment]
> Osmose aussi concerné, et toujours pas en v2 [osm-fr/osmose-frontend#481](https://github.com/osm-fr/osmose-frontend/issues/481)

C'est bon pour Osmose.

**Comment by cayenne17** [comment]
Umap a été migré sur OAuth 2.0

<br>

Dans les applis que j'ai cité ci-dessus, je vois qu'il reste encore :
- https://commerces.openstreetmap.fr/
- https://projetdumois.fr/

<br>

Il y en a que je n'ai pas cité et qui n'ont pas encore migré et qu'il m'est arrivé d'utiliser :
- OsmAND
- OSMCha
- Luke Sidewalker

**Comment by Cyrille37** [comment]
https://projetdumois.fr/ a son ticket : https://github.com/vdct/ProjetDuMois/issues/312

**Comment by cayenne17** [comment]
https://commerces.openstreetmap.fr n'a pas été encore migré.

**Comment by Jean-BaptisteC** [comment]
Le dernier service n'existe plus, je pense qu'on peut fermer le ticket

**Comment by Cyrille37** [comment]
Effectivement, le service OAtuh 1 est arrêté depuis mi-2024. Le ticket peut donc être fermé :)
